### ISSUE TITLE
Write query on more than one line

### ISSUE THREAD

**Issue by daniaalkhalil** [dbfit/dbfit#437]
we get an error in execution of the test case  when we have to write an sql queries and we put it on more Than 1line

**Comment by MMatten** [comment]
Can you please post an example of the test here?

I take it you've already tried enclosing your SQL code with `!-` and `-!`? E.g. as in https://github.com/dbfit/dbfit/blob/master/FitNesseRoot/DbFit/AcceptanceTests/JavaTests/OracleTests/FlowMode/MultiLineQueries/content.txt.

**Comment by javornikolov** [comment]
@daniaalkhalil, in general there is a trouble with multi-line mode and `<`, `>` characters. To check if that's the issue in our case: you may test what happens if you're using e.g. just `=` instead of `<=`.

**Comment by Dan-Co** [comment]
Hi @javornikolov 
We have just started looking at DBFit for Oracle.
However, we have come accross this issue for complex SQL statements containing < and >

Using the multi-line code and < and >

This issue was a while ago, are there any reccomended workarounds considering we actually need the < and > operators in our SQL?

I am tryign to create a dummy test case, but it seems it doesnt always hit the issue.

For example, both of the below actually work ok...

```
!|Query|!-
select case when sysdate < sysdate +1 then 'X' else 'Y' end as dummy
from dual 
where 1 < 2
-!|
|DUMMY|
|X|

|Ordered Query|!-
SELECT ROWNUM n FROM ( 
  SELECT 1 FROM dual 
  CONNECT BY LEVEL < 4 
)-!|
|n|
|1|
|2|
|3|
```

I will continue to try and force a test case, but it seems this is a known limitation from a few posts ive read...

**Comment by javornikolov** [comment]
Yes, this has problem with multiline queries. I'm not sure if I tried to use some define bar instead < and > characters.

**Comment by andreasmarkussen** [comment]
Workarround is to use BETWEEN
